src/components/Summary/InterviewSummaryModal/index.js

```diff
@@ -199,166 +126,78 @@ const InterviewSummaryModal = props => (
-                            >
-                              Code quality
-                            </Rate>
-                          </Col>
-                          <Col>
-                            <Rate
-                              onChange={new_val => {
-                                formdata.compeleteness = new_val;
-                                console.log(formdata);
-                              }}
-                            >
-                              Compeleteness
-                            </Rate>
-                          </Col>
-                          <Col>
-                            <Rate
-                              onChange={new_val => {
-                                formdata.hints = new_val;
-                                console.log(formdata);
-                              }}
-                            >
-                              How much hints
-                            </Rate>
-                          </Col>
-                          <Col>
-                            <Input
-                              onChange={e => {
-                                comment_name = e.target.value;
-                              }}
-                            />
-                            Please leave your name
-                          </Col>
-                          <Col>
-                            <Input
-                              onChange={e => {
-                                comment_text = e.target.value;
-                              }}
-                            />
-                            Please leave your comment
-                          </Col>
-                          <Button type="primary" htmlType="submit">
-                            Add a Score
-                          </Button>
-                        </Form>
-                      </div>
-                    );
-                  } else {
-                    questioncomment = (
+                  const questioncomment = (
+                    <>
                       <QuestionComment
                         interviewer={comment.author}
-                        questions={[questions]}
+                        questions={[questions[0]]}
                         comments={[comment]}
                       />
-                    );
-                  }
-                  comment_count = comment_count + 1;
+                    </>
+                  );
+
                   return questioncomment;
                 })}
-                <Typography.Title level={4}>
-                  Comments by questions
-                </Typography.Title>
+                <br></br>
+                {new_score}
+                <br></br>
+                <Typography.Title level={4}>Comments</Typography.Title>
                 <Row type="flex" justify="space-around">
                   {records.map(record => {
+                    let wrap_data = [];
+                    const columns = [
```

Comment:
const `columns` is a static variable. It can be put outside of `InterviewSummaryModal`, so that it won't be re-created every time render.